README.md

```diff
@@ -14,30 +14,76 @@ This README contains a high level description of contributing a SWIP, for conven
+The PR is adding at least a new placeholder file for the new proposal named `XXXX.md` to be located in the `SWIPs` subdirectory of the SWIPs repo.
 
-# SWIP Status Terms
+This placeholder file contains minimally only the the yaml header with the following fields:
 
-* **Draft** - a SWIP that is undergoing rapid iteration and changes.
-* **Last Call** - a SWIP that is done with its initial iteration and ready for review by a wide audience.
-* **Accepted** - a core SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Core Devs to decide whether to encode a SWIP into their clients as part of a mandatory update is not part of the SWIP process. If such a decision is made, the SWIP will move to final.
-* **Final (non-Core)** - a SWIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - a SWIP that the Core Devs have decided to implement and release in a future mandatory update or has already been released as such. 
-* **Deferred** - a SWIP that is not being considered for immediate adoption. May be reconsidered in the future.
+```yaml
+ ---
+title: fairx
+SWIP: XXXX
+author: Viktor Tron (@zelig), Viktor Toth (@nugaon)
+status: Draft
+---
+```
+
+- The title must be descriptive yet concise
+- SWIP is the SWIP number and must be the same as the filename
+- Authors must be specified as a comma-separated list of `real name (@github_handle)`
+- The status must be initially set to `Draft`. see below the section on status.
+
+The SWIP editors can merge this initial request to signal the SWIP worthy of recording and ready for discussion.
+
+From this moment on, the SWIP number is taken as allocated and the SWIP will permanently be part of the repo (yes even if it is deferred or rejected).
+This way looking at the repo should give the number to be used when  a new SWIP proposal is submitted. 
+
+Submitters of SWIPs are recommended to use a mnemonic feature branch name. 
+
+### Updating a SWIP
+
+A suggested edit to a SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) with a PR title in the following format: `Update SWIP-XXXX: Title of the suggested change`. 
+
+When a PR is already open for the SWIP improvments must be submitted as PRs against this feature branch. Organising reviews to these and merging them is delegated to the submitter (names in the `authors` header).
+
+Once a phase of a SWIPs is complete, authors will indicate this by setting the `ready to merge` label on the open PR. editors can merge the PR to master. see also section about curating SWIPs below.
+
+### Format and structure
```

Comment:
Should the template SWIP (https://github.com/ethersphere/SWIPs/blob/master/SWIPs/swip-X.md) still be used? I guess not, since there are sections and headers defined in this README.